metadata-io/src/main/java/com/linkedin/metadata/entity/cassandra/CassandraAspectDao.java

```diff
@@ -545,8 +546,10 @@ private SimpleStatement generateSaveStatement(EntityAspect aspect, boolean inser
               .value(CassandraAspect.CREATED_ON_COLUMN, literal(aspect.getCreatedOn().getTime()))
               .value(CassandraAspect.CREATED_FOR_COLUMN, literal(aspect.getCreatedFor()))
               .value(CassandraAspect.ENTITY_COLUMN, literal(entity))
-              .value(CassandraAspect.CREATED_BY_COLUMN, literal(aspect.getCreatedBy()))
-              .ifNotExists();
+              .value(CassandraAspect.CREATED_BY_COLUMN, literal(aspect.getCreatedBy()));
+      if (updateIfCreatedOn != null && updateIfCreatedOn == 0) {
```

Comment:
What happens if `updateIfCreatedOn != null` and `updateIfCreatedOn != 0`? We maybe should be able to return a BadRequest via HTTP if so. We at least need to throw an exception back to the client.

Reply:
Show we throw an exception from CassadraDao class?

Reply:
We could, although it's possible to spot this sooner in the call stack if it makes more sense. I think we need to catch it somewhere so it's preferable before we get to Cassandra/Ebean specific implementations.

Reply:
I don't think it is possible as Cassadra implementation doesn't do the database read at all so we can't know until we execute the update query and get the response.